### DISCUSSION TITLE
Vscode i18n Ally extension

### DISCUSSION THREAD

**Discussion by mertafor** [amannn/next-intl#293] [Q&A]
Is there a way to work with i18n-ally extension of vscode? 
https://github.com/lokalise/i18n-ally

**Comment by amannn** [comment]
I was curious about this too and briefly looked into a proof-of-concept here: https://github.com/lokalise/i18n-ally/issues/820. I couldn't get the extension to work locally, if someone knows how to do this, I'd be happy to hand over the work!
